ui/simul/src/interfaces.ts

```diff
@@ -23,6 +23,7 @@ export interface SimulData {
   isRunning: boolean;
   isFinished: boolean;
   text: string;
+  estimatedStartAt: string;
```

Comment:
should be optional with `?`.

Also, not used, so not needed

Reply:
Ah, yes.

587afa0